exchanges/bybit/bybit_test.go

```diff
@@ -2794,6 +2794,8 @@ func TestGetActiveOrders(t *testing.T) {
 	var getOrdersRequestSpot = order.GetOrdersRequest{
 		Pairs:     currency.Pairs{pair},
 		AssetType: asset.Spot,
+		Side:      order.AnySide,
+		Type:      order.AnyType,
 	}
 
 	_, err = b.GetActiveOrders(context.Background(), &getOrdersRequestSpot)
```

Comment:
Please update the remaining `order.GetOrdersRequest` scenarios to have `Side` and `Type`